rllib/utils/test_utils.py

```diff
@@ -461,6 +461,13 @@ def _test(
                     timestep = random.randint(0, 100000)
                     for unsquash in [True, False, None]:
                         for clip in [False] if unsquash else [True, False, None]:
+                            print("-" * 80)
```

Comment:
let's leave these prints here? it helps debugging later.

Reply:
won't you see a lot of prints this way?
is this only used in test?

Reply:
it's only for tests.